code/modules/power/supermatter/supermatter.dm

```diff
@@ -128,6 +128,9 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/temp_factor = 30
 	var/power = 0
 
+	/// Rad Modifier - Also Increases Power
```

Comment:
no it doesnt

Reply:
Yes it does? More rads = More Power? I tested this and it works as intended.